modules/gateway/consts.go

```diff
@@ -68,6 +77,14 @@ var (
 		Dev:      15,
 		Testing:  10,
 	}).(int)
+
+	// quickPruneListLen defines the number of nodes that the gateway must have
+	// to be pruning nodes quickly from the node list.
+	quickPruneListLen = build.Select(build.Var{
+		Standard: 250,
+		Dev:      40,
+		Testing:  20,
+	}).(int)
```

Comment:
it has to do with how constants are handled. If you have a declaration like this:
```go
const foo uint64 = 7
```
Then Go knows that `7` is supposed to interpreted as a `uint64`. But if you write this:
```go
var foo interface{} = 7
```
...the compiler just has to guess. And for integers, it always guesses `int`.

Since the fields of `build.Var` are all `interface{}`s, at compile time the integers are converted to `int` before they are assigned. Then we do a type assertion on the return value of `build.Select`. In this case, it returns an `int`, so the assertion works. But if we changed it to `uint64`, it would fail.
This is one of the unfortunate downsides of the `build.Select`/`build.Var` design.